sources/install/package_osint.sh

```diff
@@ -140,8 +140,8 @@ function install_infoga() {
     git -C /opt/tools/ clone --depth 1 https://github.com/m4ll0k/Infoga
     find /opt/tools/Infoga/ -type f -print0 | xargs -0 dos2unix
     cd /opt/tools/Infoga
-    python3 -m venv ./venv
-    ./venv/bin/python3 -m pip install .
+    python2 -m virtualenv ./venv
+    ./venv/bin/python2 -m pip install .
```

Comment:
Why switch from python3 to python2 ?

Reply:
python3.11 error :
```
./venv/bin/python3 -m pip install .
Processing /opt/tools/Infoga
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      /opt/tools/Infoga/venv/lib/python3.11/site-packages/setuptools/_distutils/dist.py:265: UserWarning: Unknown distribution option: 'console'
        warnings.warn(msg)
      error: Multiple top-level packages discovered in a flat-layout: ['lib', 'recon', 'screen'].

      To avoid accidental inclusion of unwanted files or directories,
      setuptools will not proceed with this build.

      If you are trying to create a single distribution with multiple packages
      on purpose, you should not rely on automatic discovery.
      Instead, consider the following options:

      1. set up custom discovery (`find` directive with `include` or `exclude`)
      2. use a `src-layout`
      3. explicitly set `py_modules` or `packages` with a list of names

      To find more information, look for "package discovery" on setuptools docs.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details. 
```